extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -484,6 +484,8 @@ export const createNewLocalAppSettingWithIcon = `$(add) ${createNewLocalAppSetti
 export const valueMustNotBeEmpty = localize('valueMustNotBeEmpty', "Value must not be empty");
 export const enterConnectionStringSettingName = localize('enterConnectionStringSettingName', "Enter connection string setting name");
 export const enterConnectionString = localize('enterConnectionString', "Enter connection string");
+export const saveChangesInFile = localize('saveChangesInFile', "There are unsaved changes in the current file. Save before continuing?");
```

Comment:
The text seems to imply that you can continue even if you don't save - but the code will just exit early in that case. Maybe something like `There are unsaved changes in the current file. Save now?` @dzsquared thoughts?